pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewApplicator.java

```diff
@@ -568,6 +570,7 @@ public RowElement(Column columnInfo, List<IBase> values) throws ViewApplicationE
     }
 
     // Convenience function
+    @Nullable
```

Comment:
Hmm, this might be a little bit tricky as a column `name` is not really `Nullable` (I know we have defined it that way right now but in the actual spec it is a required field and we have a check for it [here](https://github.com/google/fhir-data-pipes/blob/5be243eefa7f762264856fc9215d89833bba32fc/pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewDefinition.java#L179)). It is okay to leave the `@Nullable` but in general I like to prevent passing nulls around (and accounting for them) as much as possible.